docs/streaming-custom-receivers.md

```diff
@@ -257,54 +257,61 @@ The following table summarizes the characteristics of both types of receivers
-See [ActorWordCount.scala](https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala) for an end-to-end example.
-</div>
-<div data-lang="java" markdown="1">
+	You need to extend [`ActorReceiver`](api/scala/index.html#org.apache.spark.streaming.akka.ActorReceiver)
+	so as to store received data into Spark using `store(...)` methods. The supervisor strategy of
+	this actor can be configured to handle failures, etc.
 
-Custom [Akka UntypedActors](http://doc.akka.io/docs/akka/2.3.11/java/untyped-actors.html) can also be used to
-receive data. Extending [`JavaActorReceiver`](api/scala/index.html#org.apache.spark.streaming.akka.JavaActorReceiver)
-allows received data to be stored in Spark using `store(...)` methods. The supervisor strategy of
-this actor can be configured to handle failures, etc.
+		class CustomActor extends ActorReceiver {
+		  def receive = {
+		    case data: String => store(data)
+		  }
+		}
 
-{% highlight java %}
+		// A new input stream can be created with this custom actor as
+		val ssc: StreamingContext = ...
+		val lines = AkkaUtils.createStream[String](ssc, Props[CustomActor](), "CustomReceiver")
 
-class CustomActor extends JavaActorReceiver {
-  @Override
-  public void onReceive(Object msg) throws Exception {
-    store((String) msg);
-  }
-}
+	See [ActorWordCount.scala](https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala) for an end-to-end example.
+	</div>
+	<div data-lang="java" markdown="1">
 
-// A new input stream can be created with this custom actor as
-JavaStreamingContext jssc = ...;
-JavaDStream<String> lines = AkkaUtils.<String>createStream(jssc, Props.create(CustomActor.class), "CustomReceiver");
+	You need to extend [`JavaActorReceiver`](api/scala/index.html#org.apache.spark.streaming.akka.JavaActorReceiver)
+	so as to store received data into Spark using `store(...)` methods. The supervisor strategy of
+	this actor can be configured to handle failures, etc.
 
-{% endhighlight %}
+		class CustomActor extends JavaActorReceiver {
+		  @Override
+		  public void onReceive(Object msg) throws Exception {
+		    store((String) msg);
+		  }
+		}
 
-See [JavaActorWordCount.scala](https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/streaming/JavaActorWordCount.scala) for an end-to-end example.
-</div>
-</div>
+		// A new input stream can be created with this custom actor as
+		JavaStreamingContext jssc = ...;
+		JavaDStream<String> lines = AkkaUtils.<String>createStream(jssc, Props.create(CustomActor.class), "CustomReceiver");
+
+	See [JavaActorWordCount.scala](https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/streaming/JavaActorWordCount.scala) for an end-to-end example.
+	</div>
+	</div>
+
+3. **Deploying:** As with any Spark applications, `spark-submit` is used to launch your application.
```

Comment:
To make the list number here display 3, I changed some indents in `programming`. Please use https://github.com/apache/spark/pull/10856/files?w=1 to review the content.